frontend/src/authschemes/webauthn/services.ts

```diff
@@ -72,3 +72,8 @@ export async function listWebauthnCredentials(): Promise<CredentialList> {
 export async function deleteWebauthnCredential(i: { credentialName: string }): Promise<CredentialList> {
   return await req('DELETE', `/auth/webauthn/credential/${i.credentialName}`)
 }
+
+// TODO TN what do other PUTs return?
```

Comment:
> what do other PUTs return?

Looks like either nothing (more common) or the revised entity (less common), at least for this project. I think this is fine as void.

Reply:
removed